test/xspec-prefix-conflict-import.xspec

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<t:description
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
+	stylesheet="xspec-xmlns.xsl"
+	xslt-version="2.0">
```

Comment:
This `@xslt-version` is the same as the default. Probably it can be removed?

Reply:
Yes, I'll remove it in my next commit. While it's a contrast with another test that requires 3.0, the attribute is not needed.

Reply:
Thanks for clarifying it. I thought it might possibly be a typo of `3.0`. So I guess better to be removed.

Reply:
Resolved in acab3d491188b8ba22921e61c7d6a006f4eacfda